app/classifier/annotation-renderer/svg.jsx

```diff
@@ -189,7 +191,7 @@ export default class SVGRenderer extends React.Component {
                 />
               </Draggable>
             )}
-
+            <ProgressMarker />
```

Comment:
One thought: you could conditionally push the `ProgressMarker` into the `children` array, when the subject hooks are processed.